src/ocean/core/BitArray.d

```diff
@@ -189,13 +191,8 @@ struct BitArray
     }
     body
     {
-        size_t mDim=len/32;
-        ptr[0..mDim] = rhs.ptr[0..mDim];
-        int rest=cast(int)(len & cast(size_t)31u);
```

Comment:
Yes, sorry, you're right, my other comment has this fixed: https://github.com/sociomantic-tsunami/ocean/pull/56#issuecomment-288678541